src/components/textarea/__tests__/TextArea.Test.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { userEvent } from '@storybook/testing-library';
+
+import Textarea from '../index';
+
+const renderWrapper = ({
+  name = 'Textarea',
+  autoFocus = false,
+  isDisabled = false,
+  onChange = jest.fn(),
+  onFocus = jest.fn(),
+  onBlur = jest.fn(),
+  placeholder = 'Placeholder',
+} = {}) =>
+  render(
+    <Textarea
+      name={name}
+      autoFocus={autoFocus}
+      isDisabled={isDisabled}
+      onChange={onChange}
+      placeholder={placeholder}
+      onFocus={onFocus}
+      onBlur={onBlur}
+    />
+  );
+
+describe('<Textarea>', () => {
+  it('is called when focusing the textarea', () => {
```

Comment:
Can you take a look at the test? It looks like we could either rephrase the names or group them under describe. So something like either:

```
it('onFocus is called when focusing the textarea', () => {
```

or 

```
describe('onFocus', () => {
  it('is called when focusing the textarea', () => {
```